examples/systemcalls/systemcalls.c

```diff
@@ -16,8 +16,21 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
+    //Check for NULL
+    if(cmd == NULL)
+    return false;
+
+    //Call system function with cmd in parameter
+    int syscmd = system(cmd);
+    
+    //Check return from system call
+    WIFEXITED(syscmd);
+
+    if (syscmd == true)
+    	return true;
```

Comment:
@cchoi22915 I'm surpised this works.  I think you should invert this logic and if system returns 0 it would mean both the system() call and child shell completed with success status.